### PR TITLE
Babylon naval removal

### PR DESCRIPTION
**Pull request by Danarca**

### CODE REVIEW

File: common/journal_entries/babylon_bulgaria_je.txt
Comment (Danarca) [inline common/journal_entries/babylon_bulgaria_je.txt:1]: Don't know why there's something changed here :s

File: common/buildings/babylon_military.txt [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
 	levels_per_mesh = 0
-
-	can_build_government = {
+	possible = {
+		is_capital = yes
 		error_check = {
+			severity = potential
 			is_capital = yes
 		}
 	}
```
Comment (HiddeLekanne) [inline common/buildings/babylon_military.txt]: does this mean that private construction can build it?

File: common/journal_entries/babylon_bulgaria_je.txt
Comment (HiddeLekanne) [reply common/journal_entries/babylon_bulgaria_je.txt:1]: formatting

File: common/buildings/babylon_military.txt (hunk 1 above)
Comment (Danarca) [reply common/buildings/babylon_military.txt]: Fixed :)

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**